Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -104,6 +116,26 @@ internal static class ProfileManager
             return profileProcessResponse;
         }
 
+        internal static int GetIisProfilingTimeout()
+        {
+            int timeout = -1;
+
+            string iisProfilingTimeoutInSeconds = System.Environment.GetEnvironmentVariable("APPSETTING_IIS_PROFILING_TIMEOUT_IN_SECONDS");
+
+            if (!string.IsNullOrEmpty(iisProfilingTimeoutInSeconds))
+            {
+                if (Int32.TryParse(iisProfilingTimeoutInSeconds, out timeout))
+                {
+                    if (timeout>(60*15)) // making sure that no configures it more than 15 minutes
```

Comment:
not following...This is not actually a multiple of _profliingTimeout. This is an upper limit of _profilingTimeout....Looks like your comment is missing some characters...